website/docs/src/luigi-config/extended/routing.js

```diff
@@ -6,7 +6,7 @@ class Routing {
    * For hash routing, set to true and run `npm run start`
    * For path routing, set to false and run `npm run start`
    */
-  useHashRouting = false;
+  useHashRouting = true;
```

Comment:
The docs should run with path routing?
```suggestion
  useHashRouting = false;
```